src/components/Sidebar.tsx

```diff
@@ -113,6 +113,10 @@ export default function Sidebar(props: sidebarProps): JSX.Element {
       name: '3. Dereferencing Pointers',
       link: PageURLs.EXERCISE_3,
     },
+    {
```

Comment:
Can you move this outside of the list?
The conclusion is not an exercise, so it shouldn't go in the exercises dropdown.
<img width="352" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/2f24d58b-c7d8-4fb1-8e62-e12379f3f8ab">